### DISCUSSION TITLE
Configuring routes in auth0

### DISCUSSION THREAD

**Discussion by BispensGipsGebis** [quasarframework/quasar#10081] [CLI - BEX mode]
Hi wonderful community,

I'm, fairly new to quasar and am trying to set up authentication with Auth0. I've managed to set up the boot, but I fail to understand where/how to set up the necessary routes.

Trying to follow the guidelines for .vue on Auth0, but I fail to get it to make sense.

Any pointers would be highly appreciated.

Best,
Rune

**Comment by ontwikkelfabriek** [comment]
@BispensGipsGebis 

I found this repos that with a Quasar app that uses 0auth:
https://github.com/34fame/quasar-mobile-auth0-firebase
https://wcomnisky.github.io/quasar-hello-auth/
